src/client/styles/objects/_reading-width.scss

```diff
@@ -1,3 +1,4 @@
 .reading-width {
   @include reading-width;
+  max-width: none !important;
```

Comment:
I agree with you in most points. The problem was some divs just stopping at max-width some others are fluid. It looks a bit weird.
It would take a bit of investigation to do it properly. Hopefully we will have a proper front-end developer to fix this issue.
It's not extremely important to use this quick fix right now. We can do it properly later.